stdlib/public/TensorFlow/CompilerRuntime.swift

```diff
@@ -1183,3 +1178,8 @@ public func _CreateTensorHandleFromCTensorHandle(
   return TensorHandle<Float>(owning: ownedCHandle)
 }
 
+@inlinable
+@_cdecl("_swift_tfc_CheckOk")
+public func _CheckOk(_ s: CTFStatus) {
```

Comment:
All our previous existing exported runtime functions have prefix “_TFC” in the Swift name. These should be updated for consistency. For example, this function should be “_TFCCheckOK”.

Reply:
Done.